TLM/TLM/UI/MainMenu/TimedTrafficLightsButton.cs

```diff
@@ -1,12 +1,12 @@
-﻿namespace TrafficManager.UI.MainMenu {
+namespace TrafficManager.UI.MainMenu {
     using State;
 
     public class TimedTrafficLightsButton : MenuToolModeButton {
         protected override ToolMode ToolMode => ToolMode.TimedLightsSelectNode;
 
         protected override ButtonFunction Function => ButtonFunction.TimedTrafficLights;
 
-        public override string Tooltip => Translation.Menu.Get("Tooltip:Timed traffic lights");
+        public override string Tooltip => Translation.Menu.Get("Tooltip:Timed traffic lights") + "\n" + Translation.Menu.Get("Tooltip.Keybinds:Quick setup");
```

Comment:
Tooltip key is overly generic "Tooltip.Keybinds:Quick setup" should be changed to something more specific like "Tooltip.Keybinds:Timed TL quick setup"